### PR TITLE
Geearl/7564 additional owners

### PR DESCRIPTION
**Pull request by georearl**
This PR adds a value ENTRA_OWNERS to the local.env.example file. It allows you to enter a comma separated list of email addresses of id's within your tenant that you wish to be marked as owners for the Azure Entra objects. The default is an empty string

### CODE REVIEW

**Review by KronemeyerJoshua — DISMISSED** [review]
Looks good besides typo

File: docs/deployment/deployment.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ DEFAULT_LANGUAGE | Yes | Use the parameter to specify the matching ENV file loca
 ENABLE_CUSTOMER_USAGE_ATTRIBUTION <br>CUSTOMER_USAGE_ATTRIBUTION_ID | No | By default, **ENABLE_CUSTOMER_USAGE_ATTRIBUTION** is set to `true`. The CUA GUID which is pre-configured will tell Microsoft about the usage of this software. Please see [Data Collection Notice](/README.md#data-collection-notice) for more information. <br/><br/>You may provide your own CUA GUID by changing the value in **CUSTOMER_USAGE_ATTRIBUTION_ID**. Ensure you understand how to properly notify your customers by reading <https://learn.microsoft.com/en-us/partner-center/marketplace/azure-partner-customer-usage-attribution#notify-your-customers>.<br/><br/>To disable data collection, set **ENABLE_CUSTOMER_USAGE_ATTRIBUTION** to `false`.
 ENABLE_DEV_CODE | No | Defaults to `false`. It is not recommended to enable this flag, it is for development testing scenarios only.
 APPLICATION_TITLE | No | Defaults to "". Providing a value for this parameter will replace the Information Assistant's title in the black banner at the top of the UX.
+ENTRA_OWNERS | No | Defailts to "". Additional user id's you wish to assign as owners of created Azure Entra objects by way of assign to a security group.
```
[the remark this answers is not in the document]
Comment (georearl) [reply docs/deployment/deployment.md]: fixed

File: scripts/inf-create.sh [hunk 2]
```diff
@@ -60,6 +60,20 @@ then
     az account set -s "$ARM_SUBSCRIPTION_ID"
 fi
 
+
+# prepare vars for the users you wish to assign to the security group
+object_ids=()
+# Remove spaces from the comma-separated string
+ENTRA_OWNERS=$(echo "$ENTRA_OWNERS" | tr -d ' ')
+IFS=',' read -ra ADDR <<< "$ENTRA_OWNERS"
+for user_principal_name in "${ADDR[@]}"; do
```
Comment (dayland) [inline scripts/inf-create.sh:69]: What happens if an invalid email address is added?

**Review by dayland — APPROVED** [review]

File: scripts/inf-create.sh (hunk 2 above)
Comment (georearl) [reply scripts/inf-create.sh:69]: This now accomodates a bad email address, so it isn't added to the array of id's passed to TF. It also handles spaces between commas/email addresses

**Pull request merged**